src/argilla_sdk/records/_dataset_records.py

```diff
@@ -338,36 +342,49 @@ def from_json(self, path: Union[Path, str]) -> "DatasetRecords":
             DatasetRecords: The DatasetRecords object created from the disk path.
 
         """
-        with open(path, "r") as f:
-            records = json.load(f)
-        self.update(records=[_dict_to_record(record) for record in records])
+        records = JsonIO._records_from_json(path=path)
+        return self.update(records=records)
         return self
```

Comment:
```suggestion
        return self.update(records=records)
```

Reply:
I'm a bit confused by the current return values of `update` and `add` because they return the records themselves rather than self. Do you think all should return `List[Record]`.